### PR TITLE
Make final fixes

### PR DESCRIPTION
**Pull request by carpecodeum**
# Description
Final fixes in web application.

Fixes #424 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:

- Firmware version:
- Hardware:
- Toolchain:
- SDK:

Also, include screenshots of app for the verification and reviewing purpose.

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by ivantha** [comment]
@carpecodeum Did you reformat the codebase as well?

**Comment by carpecodeum** [comment]
> @carpecodeum Did you reformat the codebase as well?

I am currently working on that will update you asap :)

File: labellab-client/src/components/login/ResetPassword.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ class ResetPassword extends Component {
       updated:false
     };
   }
-
+F
```
Comment (hiranthaR) [inline labellab-client/src/components/login/ResetPassword.js:21]: there is a additional F letter i think :slightly_smiling_face:

Comment (carpecodeum) [reply labellab-client/src/components/login/ResetPassword.js]: done!

**Review by hiranthaR — COMMENTED** [review]
lgtm :slightly_smiling_face:

**Comment by carpecodeum** [comment]
@ivantha can we merge this?

**Pull request merged**